### DISCUSSION TITLE
希望加入每日定时运行功能

### DISCUSSION THREAD

**Discussion by Diorite9248** [moesnow/March7thAssistant#80] [Q&A] [answered]
因为客观条件不允许24小时开机，设置的每日早上6:30启动应用程序，但是貌似无法自动“完整运行”，希望加入自定义时间启动“完整运行”及其他各项任务

**Comment by moesnow** [comment, marked as answer]
如果需要使用 任务计划程序 定时运行或直接执行 完整运行，可以使用终端图标的 March7th Assistant.exe，而不是三月七图标的 March7th Launcher.exe

**Reply by Diorite9248** [reply]
谢谢作者，原来如此
